### PR TITLE
MGDOBR-980: [UI] Add source definition test

### PR DESCRIPTION
**Pull request by sjamboro**
https://issues.redhat.com/browse/MGDOBR-980

Create new test suite for the configuration of Processor's source.

### CODE REVIEW

File: cypress/integration/SourceTest.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+    it("The default state of the configuration", () => {
+      cy.ouiaId("missing-sources", "PF4/TextInput")
+        .scrollIntoView()
+        .should("be.visible")
+        .should("be.disabled");
+      cy.ouiaId("configuration").should("not.exist");
+    });
+
+    describe("Send to Slack", () => {
```
Comment (jomarko) [inline cypress/integration/SourceTest.ts:47]: minor, I think we should name `Slack Source`

**Review by jomarko — APPROVED** [review]
Thank you @sjamboro looks good to me.

File: cypress/integration/SourceTest.ts (hunk 1 above)
Comment (sjamboro) [reply cypress/integration/SourceTest.ts]: @jomarko fixed

**Review by ChughShilpa — APPROVED** [review]
LGTM. Thanku @sjamboro for PR.

File: cypress/integration/SourceTest.ts [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+              .should("not.exist");
+            cy.ouiaId("kafka_topic", "config-parameter")
+              .should("be.visible")
+              .find("div.pf-m-error")
+              .should("not.exist");
+          });
+      });
+    });
+
+    describe("Dummy Source", () => {
```
Comment (kelvah) [inline cypress/integration/SourceTest.ts:111]: Hi @sjamboro, could you please clarify why you introduced a new dummy source schema? Thanks

Comment (sjamboro) [reply cypress/integration/SourceTest.ts:111]: @kelvah the reason is that all sources contains lots of parameters and all of them contain token as required. The action list contains "Send to bridge" with has only one optional parameter. The dummy source is ideal for mock tests to check if number of parameters was changed.

Comment (kelvah) [reply cypress/integration/SourceTest.ts:111]: It looks a bit overkill to me, but it seems it makes life easier for testing. Approving then. Thank you.

**Review by kelvah — APPROVED** [review]

**Pull request merged**